node/tests/SharedTests.ts

```diff
@@ -873,6 +873,60 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `lset with nonexisting key`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient) => {
+                const key = "test_key";
+                const non_existing_key = "nonexisting";
```

Comment:
I know some of the other tests have this same variable name, but typescript uses camel-case, so can we switch to align with that?
```suggestion
                const nonExistingKey = "nonexisting";
```